os/arch/arm/src/stm32l4/stm32l4_rtc_lowerhalf.c

```diff
@@ -697,32 +698,28 @@ static int stm32l4_cancelperiodic(FAR struct rtc_lowerhalf_s *lower, int id)
  ****************************************************************************/
 
 /****************************************************************************
- * Name: stm32l4_rtc_lowerhalf
+ * Name: stm32l4_rtc_initialize
  *
  * Description:
- *   Instantiate the RTC lower half driver for the STM32.  General usage:
- *
- *     #include <tinyara/timers/rtc.h>
- *     #include "stm32l4_rtc.h>
- *
- *     struct rtc_lowerhalf_s *lower;
- *     lower = stm32l4_rtc_lowerhalf();
- *     rtc_initialize(0, lower);
+ *   Register the RTC driver to provide RTC functionality
  *
  * Input Parameters:
  *   None
  *
  * Returned Value:
- *   On success, a non-NULL RTC lower interface is returned.  NULL is
- *   returned on any failure.
+ *   None
  *
  ****************************************************************************/
 
-FAR struct rtc_lowerhalf_s *stm32l4_rtc_lowerhalf(void)
+void stm32l4_rtc_initialize(void)
 {
-  nxsem_init(&g_rtc_lowerhalf.devsem, 0, 1);
+	struct rtc_lowerhalf_s *lower = (struct rtc_lowerhalf_s *)&g_rtc_lowerhalf;
+	int ret;
 
-  return (FAR struct rtc_lowerhalf_s *)&g_rtc_lowerhalf;
+	ret = rtc_initialize(0, lower);
```

Comment:
May be we can keep the original comment:

/* Bind the lower half driver and register the combined RTC driver	
 * as /dev/rtc0	
 */

Reply:
Added.